src/main/kotlin/slack/cli/sarif/SarifUtil.kt

```diff
@@ -122,7 +122,16 @@ internal fun List<SarifSchema210>.merge(
     flatMap { it.runs.single().results.orEmpty() }
       // Some projects produce multiple reports for different variants, so we need to
       // de-dupe.
-      .distinct()
+      // Using the default distinct() function leaves duplicates, so using a custom selector
+      .distinctBy {
+        it.ruleID +
+          it.message +
```

Comment:
Message is an object with a bunch of stuff, may want to use `message.text` to limit it